runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -1345,6 +1340,12 @@ public synchronized CorfuRuntime connect() {
 
         log.info("connect: runtime parameters {}", getParameters());
 
+        // Start file watcher on Ssl certs
+        if (!sslCertWatcher.isPresent()) {
+            log.info("connect: Initializing sslCertWatcher.");
+            sslCertWatcher = getSslCertWatcher();
+        }
+
```

Comment:
Can you add a test to verify that thread's life-cycle ?

Reply:
Added `testFileWatcherLifeCycle()` now, Thanks!